python/HeaderService/hutils.py

```diff
@@ -150,12 +150,14 @@ def write_header_string(arg):
     return
 
 
-def get_date_utc(time=None, format='isot'):
+def get_date_utc(timeStamp=None, format='isot'):
```

Comment:
Please document your APIs using standard DM numpydoc.

Reply:
Added to the API